From 77421020c9ab36c1e701901d7e72747ca98d3133 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Sat, 22 Dec 2018 19:49:47 +0900
Subject: [PATCH 12/34] Revert "udevd: configure a child process name for
 worker processes"

This reverts commit 49f3ee7e74c714f55aab395c080b1099fc17f7fd.

(cherry picked from commit ff86c92e3043f71fc801cf687600a480ee8f6778)
---
 src/udev/udevd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/udev/udevd.c b/src/udev/udevd.c
index fb8724ea87..ec77bd4a71 100644
--- a/src/udev/udevd.c
+++ b/src/udev/udevd.c
@@ -534,7 +534,7 @@ static int worker_spawn(Manager *manager, struct event *event) {
         if (r < 0)
                 return log_error_errno(r, "Worker: Failed to enable receiving of device: %m");
 
-        r = safe_fork("(worker)", FORK_DEATHSIG, &pid);
+        r = safe_fork(NULL, FORK_DEATHSIG, &pid);
         if (r < 0) {
                 event->state = EVENT_QUEUED;
                 return log_error_errno(r, "Failed to fork() worker: %m");
-- 
2.20.1

